src/js/visual/image.js

```diff
@@ -119,6 +119,7 @@ ripe.Image.prototype.init = function() {
         this.options.frameValidator === undefined
             ? (...args) => this.owner.hasFrame(...args)
             : this.options.frameValidator;
+    this._options = this.options.options || null;

```

Comment:
on second thought maybe use `composeOptions` following the naming of `composeLogic` what do you think?